ui/imports/shared/status/StatusImageModal.qml

```diff
@@ -61,7 +61,7 @@ Popup {
             anchors.fill: parent
             acceptedButtons: Qt.LeftButton | Qt.RightButton
             onClicked: {
-                root.clicked(mouse.button)
+                root.clicked(mouse)
```

Comment:
This is a breaking change. Please adjust your commit message to make this clear.
Also need to ensure this is adjusted everywhere

Reply:
From my search it is used only in AppMain.qml.

Reply:
Done with commit message - please check again